debian/control

```diff
@@ -0,0 +1,16 @@
+Source: youtube-dl-gui
+Section: web
+Priority: optional
+Maintainer: nodiscc <nodiscc@gmail.com>
+Build-Depends: debhelper (>= 9), python-all
```

Comment:
You also need to add **dh-python** and **python-wxgtk3.0**.

Reply:
The proper way to handle this would be building a debian package for twodict. I'll try to do it when I resume work on this. If everything fails, depending on `python-pip` and installing `twodict` in a postinstall script would also work.